README.md

```diff
@@ -16,6 +16,8 @@ some hard-coded routes.
 |                       ||https://www.gov.uk/government/calls-for-evidence/safety-of-journalists-call-for-evidence|
 |                       ||https://www.gov.uk/government/calls-for-evidence/review-of-civil-legal-aid-call-for-evidence|
 |Case studies           |[case_study](https://docs.publishing.service.gov.uk/content-schemas/case_study.html)|https://www.gov.uk/government/case-studies/aiding-capability-decision-making-for-the-royal-navy|
+|Consultation   |schema: [consultation](https://docs.publishing.service.gov.uk/content-schemas/consultation.html)|https://www.gov.uk/government/calls-for-evidence/credit-union-common-bond-reform|
```

Comment:
the first example here appears to be a call for evidence? (second one is correct)